src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -263,6 +265,84 @@ public class OfflineMigrationValidation
             public string ErrorMessage { get; set; }
         }
 
+        [RavenAction("/studio-tasks/periodic-backup/test-credentials", "POST", AuthorizationStatus.ValidUser)]
```

Comment:
this endpoint doesn't have any access to private data (just contacting AWS, Azure... with the provided credentials)